spring/spring-boot-4/it-tests/session-tests/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-spring-boot4-starter-parent</artifactId>
+      <version>16.0.0-SNAPSHOT</version>
+      <relativePath>../../pom.xml</relativePath>
+   </parent>
+
+   <name>Infinispan Spring Boot 4 Starter Session IT</name>
+   <artifactId>infinispan-spring-boot4-starter-session-tests</artifactId>
+
+   <dependencies>
+      <dependency>
+         <groupId>org.infinispan</groupId>
+         <artifactId>infinispan-spring-boot4-starter-embedded</artifactId>
+      </dependency>
+
+      <dependency>
+         <groupId>org.infinispan</groupId>
+         <artifactId>infinispan-spring-boot4-starter-remote</artifactId>
+      </dependency>
+
+      <dependency>
+         <groupId>org.springframework.boot</groupId>
+         <artifactId>spring-boot-starter-web</artifactId>
+      </dependency>
+
+      <dependency>
+         <groupId>org.springframework.session</groupId>
+         <artifactId>spring-session-core</artifactId>
```

Comment:
Shouldn't that be `org.springframework.boot:spring-boot-session-X` ? With the upgrade you get no auto-configuration for Spring Session unless you opt-in for it.